go-controller/pkg/ovn/gress_policy_test.go

```diff
@@ -166,3 +170,30 @@ func TestGetL4Match(t *testing.T) {
 		assert.Equal(t, tc.expected, l4Match)
 	}
 }
+
+var _ = ginkgo.Describe("gressPolicy", func() {
+
+	var previouseIPV4Mode bool
+
+	ginkgo.BeforeEach(func() {
+		previouseIPV4Mode = config.IPv4Mode
+		config.IPv4Mode = true
+	})
+
+	ginkgo.AfterEach(func() {
+		config.IPv4Mode = previouseIPV4Mode
+	})
+
+	ginkgo.It("should manage deletiong of the same IP address from non owning Pod", func() {
+		asFactory := addressset.NewFakeAddressSetFactory()
+		sut := newGressPolicy(knet.PolicyTypeIngress, 0, "ns1", "pol1")
+		sut.ensurePeerAddressSet(asFactory)
+
+		p1 := newPod("ns1", "pod1", "node1", "10.10.10.10")
+		p2 := newPod("ns1", "pod2", "node1", "10.10.10.10")
+		sut.addPeerPods(p1)
+		sut.deletePeerPod(p2)
```

Comment:
maybe we can test a bit more complicated case, like
```
sut.addPeerPod(p2)
sut.addPeerPods(p1)
sut.deletePeerPod(p2)
// check ip is still in place
sut.deletePeerPod(p1)
// check ip is removed
```
also, what does `sut` mean? can't decypher :)

Reply:
It means [System Under Test](https://en.wikipedia.org/wiki/System_under_test). I discovered it while reading some [blog post](https://testing.googleblog.com/2021/03/), and it should make more clear what the code is supposed to test. But if it doesn't help, let's get rid of it :wink:

Reply:
I think in general than naming makes sense, but I find it a bit confusing when only 1 test is using it.